ManagementSystem/index.js

```diff
@@ -0,0 +1,70 @@
+const prompt = require('prompt-sync')();
+const PhoneBook = require('./phoneBook');
+const { PersonalContact, BusinessContact } = require('./Contact');
+const { ContactNotFoundError, PhoneBookError } = require('./Errors');
+
+const phoneBook = new PhoneBook();
+
+while (true) {
+    console.log("\nPhone Book Management System");
+    console.log("1. Add Contact");
+    console.log("2. Update Contact");
+    console.log("3. Delete Contact");
+    console.log("4. Display Contacts");
+    console.log("5. Save and Exit");
+
+    const choice = prompt("Enter your choice: ");
+
+    try {
+        if (choice === '1') {
+            const type = prompt("Enter type (Personal/Business): ").toLowerCase();
```

Comment:
Can create a function for user input instead of having it all inside conditional statements

Reply:
Resolved